sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -280,6 +295,23 @@ def tune(
                 + exec_script
             )
 
+        resources = client.V1ResourceRequirements()
+        if isinstance(resources_per_trial, dict):
+            requests = {
+                "cpu": "200m",
+                "memory": "256Mi",
+            }
+            if "gpu" in resources_per_trial:
+                resources_per_trial["nvidia.com/gpu"] = resources_per_trial.pop("gpu")
+            requests.update(resources_per_trial)
+
+            resources = client.V1ResourceRequirements(
+                requests=requests,
+                limits=requests,
+            )
+        else:
+            resources = resources_per_trial
+
```

Comment:
I think, we can simplify this.
Just re-configure `resources_per_trial` variable if `isinstance(resources_per_trial, dict):`
After that, just assign this variable to Pod Template Container.
In that case, we can avoid `else` statement.
Does it make sense @droctothorpe ?

Reply:
Not sure I follow, sorry, @andreyvelich! Mind clarifying?

Reply:
Do you mean something like this? i.e. reassign `resources_per_trial` and eliminate the `else`?

```python
        resources = client.V1ResourceRequirements()
        if isinstance(resources_per_trial, dict):
            requests = {
                "cpu": "200m",
                "memory": "256Mi",
            }
            if "gpu" in resources_per_trial:
                resources_per_trial["nvidia.com/gpu"] = resources_per_trial.pop("gpu")
            requests.update(resources_per_trial)

            resources_per_trial = client.V1ResourceRequirements(
                requests=requests,
                limits=requests,
            )
```

Reply:
Yeah, exactly!

After that, we can just do this:
```python
client.V1Container(
  ...,
  resources= resources_per_trial,
)
                          
```